components/renku_data_services/notifications/api.spec.yaml

```diff
@@ -0,0 +1,99 @@
+openapi: 3.0.2
+info:
+  title: Renku Data Services API
+  description: |
+    A service that allows alerts to be sent to users.
+  version: v1
+servers:
+  - url: /api/data
+paths:
+  /alerts:
+    post:
+      summary: Send an alert to a user.
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: "#/components/schemas/AlertPost"
+      responses:
+        "201":
+          description: Alert successfully created
+          content:
+            "application/json":
+              schema:
+                $ref: "#/components/schemas/Alert"
+      tags:
+        - alerts
+    get:
+      summary: Retrieve all active alerts for a user.
+      responses:
+        "200":
+          description: A list of active alerts for the user.
+          content:
+            "application/json":
+              schema:
+                $ref: "#/components/schemas/AlertList"
+      tags:
+        - alerts
+components:
+  schemas:
+    AlertList:
+      type: array
+      description: A list of alerts.
+      items:
+        $ref: "#/components/schemas/Alert"
+    AlertPost:
+      description: Data required to create an alert.
+      type: object
+      properties:
+        status:
+          $ref: "#/components/schemas/AlertStatus"
+        alerts:
+          type: array
+          description: The list of alerts.
+          items:
+            type: string
```

Comment:
Note: This payload seems too minimalist for now. It can be updated as we get a better understanding of what is an alert object.